Library/Formula/libbpg.rb

```diff
@@ -1,8 +1,8 @@
 class Libbpg < Formula
   desc "Image format meant to improve on JPEG quality and file size"
   homepage "http://bellard.org/bpg/"
-  url "http://bellard.org/bpg/libbpg-0.9.5.tar.gz"
-  sha256 "30de1d0099920e24b7c9aae4d4e6b62f446823f0a1d52eb195dfc25c662ee203"
+  url "http://bellard.org/bpg/libbpg-0.9.6.tar.gz"
+  sha256 "931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1"
```

Comment:
@Nyx0uf Did you download `http://bellard.org/bpg/libbpg-0.9.6.tar.gz` using Safari? Safari  decompresses a gzipped ( or any compressed ) file. 

```
$ shasum -a 256 libbpg-0.9.6.tar
931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1  libbpg-0.9.6.tar
```